api/MediaKeySession.json

```diff
@@ -124,10 +124,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": "13.1"
```

Comment:
@vinyldarkscratch there are still a bunch of Safari EME changes you didn't revert in https://github.com/mdn/browser-compat-data/pull/6642/commits/a277c11fe16cc326d745352f012213bfae560369. This is the first. Since it's the only remaining issue with the PR, I'll push a commit reverting the remaining Safari changes in api/MediaKey*.json, and then land this.